Library/Homebrew/diagnostic.rb

```diff
@@ -389,10 +389,10 @@ def check_homebrew_prefix
         return if HOMEBREW_PREFIX.to_s == "/usr/local"
 
         <<-EOS.undent
-          Your Homebrew is not installed to /usr/local
-          You can install Homebrew anywhere you want but some bottles (binary
-          packages) can only be used in /usr/local and some formulae (packages)
-          may not build correctly unless you install in /usr/local. Sorry!
+          Your Homebrew's prefix is not /usr/local.
```

Comment:
`Your Homebrew prefix is not...` makes sense, but `Your Homebrew's prefix` reads a bit oddly. Maybe just to me, I don't know 😄.